inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -141,6 +142,14 @@ def start(self, tls: bool = True) -> None:
         with ota_lock:
             self._perform_startup_tasks()
 
+        # Run scheduler to schedule the task
+        scheduler = APScheduler()
+        sqliteManager = SqliteManager()
```

Comment:
Let's have the database in `/var/intel-manageability/udm.db` and also have `SqliteManager` create it if it doesn't exist (first start up). Then no need to put blank `udm.db` under `fpm-template`.  (Remember to change AppArmor to be able to access the new path.)

Reply:
Location has been changed to `/var/intel-manageability/udm.db`.